Integrations/Gmail/Gmail.py

```diff
@@ -315,20 +406,99 @@ def users_to_entry(title, response):
             'Gmail': {'Address': user_data.get('primaryEmail')},
             'Group': user_data.get('kind'),
             'CustomerId': user_data.get('customerId'),
+            'Groups': user_data.get('kind'),
+            'Domain': user_data.get('primaryEmail').split('@')[1],
+            'Username': (user_data.get('name').get('givenName')
+                         if user_data.get('name') and 'givenName' in user_data.get('name') else None),
+            'OrganizationUnit': organization_format(user_data.get('organizations')),
+            'VisibleInDirectory': user_data.get('includeInGlobalAddressList'),
+
         })
-    headers = ['Type', 'ID', 'UserName',
-               'DisplayName', 'Email', 'Group', 'CustomerId']
+    headers = ['Type', 'ID', 'Username',
+               'DisplayName', 'Groups', 'CustomerId', 'Domain', 'OrganizationUnit', 'Email', 'VisibleInDirectory']
 
     return {
         'ContentsFormat': formats['json'],
         'Type': entryTypes['note'],
-        'Contents': context,
+        'Contents': response,
         'ReadableContentsFormat': formats['markdown'],
-        'HumanReadable': tableToMarkdown(title, context, headers),
+        'HumanReadable': tableToMarkdown(title, context, headers, removeNull=True),
         'EntryContext': {'Account(val.ID && val.Type && val.ID == obj.ID && val.Type == obj.Type)': context}
     }
 
 
+def autoreply_to_entry(title, response, user_id):
+    autoreply_context = []
+    for autoreply_data in response:
+        autoreply_context.append({
+            'Type': 'Google',
+            'EnableAutoReply': autoreply_data.get('enableAutoReply'),
+            'ResponseBody': autoreply_data.get('responseBodyPlainText'),
+            'ResponseSubject': autoreply_data.get('responseSubject'),
+            'RestrictToContact': autoreply_data.get('restrictToContacts'),
+            'RestrcitToDomain': autoreply_data.get('restrictToDomain'),
+
+        })
+    headers = ['Type', 'EnableAutoReply', 'ResponseBody',
+               'ResponseSubject', 'RestrictToContact', 'RestrictToDomain']
+
+    account_context = {
+        "Address": user_id,
+        "AutoReply": autoreply_context
+    }
+
+    return {
+        'ContentsFormat': formats['json'],
+        'Type': entryTypes['note'],
+        'Contents': autoreply_context,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown(title, autoreply_context, headers, removeNull=True),
+        'EntryContext': {'Account.Gmail(val.Address == obj.Address)': account_context}
```

Comment:
When we talked we decided on Account.Gmail because the auto-replay is account based.
It is changeable but I do think it makes sense it stays there

Reply:
follow the suggested change